packages/discord.js/src/index.js

```diff
@@ -177,27 +175,23 @@ exports.ReactionCollector = require('./structures/ReactionCollector');
 exports.ReactionEmoji = require('./structures/ReactionEmoji');
 exports.RichPresenceAssets = require('./structures/Presence').RichPresenceAssets;
 exports.Role = require('./structures/Role').Role;
-exports.SelectMenuBuilder = require('./structures/SelectMenuBuilder');
 exports.ChannelSelectMenuBuilder = require('./structures/ChannelSelectMenuBuilder');
 exports.MentionableSelectMenuBuilder = require('./structures/MentionableSelectMenuBuilder');
 exports.RoleSelectMenuBuilder = require('./structures/RoleSelectMenuBuilder');
 exports.StringSelectMenuBuilder = require('./structures/StringSelectMenuBuilder');
 exports.UserSelectMenuBuilder = require('./structures/UserSelectMenuBuilder');
 exports.BaseSelectMenuComponent = require('./structures/BaseSelectMenuComponent');
-exports.SelectMenuComponent = require('./structures/SelectMenuComponent');
 exports.ChannelSelectMenuComponent = require('./structures/ChannelSelectMenuComponent');
 exports.MentionableSelectMenuComponent = require('./structures/MentionableSelectMenuComponent');
 exports.RoleSelectMenuComponent = require('./structures/RoleSelectMenuComponent');
 exports.StringSelectMenuComponent = require('./structures/StringSelectMenuComponent');
 exports.UserSelectMenuComponent = require('./structures/UserSelectMenuComponent');
-exports.SelectMenuInteraction = require('./structures/SelectMenuInteraction');
 exports.ChannelSelectMenuInteraction = require('./structures/ChannelSelectMenuInteraction');
```

Comment:
I guess this TODO can be resolved now:
https://github.com/discordjs/discord.js/blob/9aed9414c080ea7cbe0cec407d9a71bef903c7d5/packages/discord.js/typings/index.d.ts#L2898